scripts/scripts.js

```diff
@@ -157,18 +154,25 @@ const loadStyle = (path) => {
   eagerLoad(marquee.querySelector('img'));
 }());
 
-const miloLibs = setLibs(LIBS);
+// eslint-disable-next-line import/prefer-default-export
+export const LIBS = (() => {
+  const { hostname, search } = window.location;
+  if (!(hostname.includes('.hlx.') || hostname.includes('local'))) return LIBS;
```

Comment:
This line is different than what we have in utils:
`if (!['.hlx.', '.stage.', 'local'].some((i) => hostname.includes(i))) return prodLibs;`